### PR TITLE
Update Docker.file to fix cloud9 issue

### PR DESCRIPTION
**Pull request by hanamz**
*Issue #86 and #88  , if available: *
https://github.com/aws-samples/aws-iot-twinmaker-samples/issues/86
https://github.com/aws-samples/aws-iot-twinmaker-samples/issues/88

*Description of changes:*
Install dependencies in Docker.file. Update IAM role used to create Cloud9 Env.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CLOUD9_SETUP.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Note: If you encounter any issues, try checking the Troubleshooting section at t
 - "Next Step" > "Configure settings" setting (can leave as defaults)
     ```
     [Environment type] Create a new EC2 instance for environment (direct access)
-    [Instance type] t2.micro (1 GiB RAM + 1 vCPU)
+    [Instance type] m4.4xlarge (64 GiB RAM + 16 vCPU)
```
Comment (johnnyw-aws) [inline CLOUD9_SETUP.md:28]: Is there a cheaper instance that can be used? (Note that m4.4xlarge is almost 100x more expensive than the previous one [[1]](https://tiny.amazon.com/wdntyn81))

Comment (hanamz) [reply CLOUD9_SETUP.md:28]: Smaller instantType will be out of disk or very slow with the latest AWS CDK and also new command we put in docker file which installed a lot of new dependencies. That might degrade the customer experiences. 
I think it is okay that we recommend the instance type that provide optimal customer experiences. They are free to choose cheaper instanceType if they want to compromise the experiences with cheaper instanceType.

**Review by johnnyw-aws — APPROVED** [review]
thanks for the fixes!

File: CLOUD9_SETUP.md (hunk 1 above)
Comment (hanamz) [reply CLOUD9_SETUP.md]: updated!

**Pull request merged**